app/contextCreator/createSolutionListPageContext.js

```diff
@@ -11,10 +11,12 @@ export const createSolutionListPageContext = solutions => ({
   pageTitle: 'All Solutions results',
   pageDescription: 'These are the Solutions on the GP IT Futures framework available from the Buying Catalogue.',
   solutions: createSolutionsContext(solutions),
+  backLinkPath: '?browse=all',

```

Comment:
I think this `backLinkPath` is slightly misleading. Since there is backLink on the solution list page and this could imply the path of the backLink on this page. 

I think naming this to something else to avoid this would be better. Upto you though, if the URL restructuring will solve this issue then happy to accept this as is.

Reply:
I have changed it for now :)